assets/js/modules/analytics-4/datastore/properties.js

```diff
@@ -240,7 +240,9 @@ const baseActions = {
 	 */
 	*matchPropertyByURL( properties, url ) {
 		const registry = yield commonActions.getRegistry();
-		const urls = ( Array.isArray( url ) ? url : [ url ] ).map( normalizeURL );
+		const urls = ( Array.isArray( url ) ? url : [ url ] )
+			.filter( ( item ) => item?.length > 0 )
```

Comment:
Maybe use `typeof item === 'string'` instead? Things that aren't strings can still have a `length` greater than 0.

Reply:
We should also add a guard to `normalizeURL` which doesn't have a safe fallback if given another type. https://github.com/google/site-kit-wp/blob/371459e27494c3c2b083249c79af390a7f1139d0/assets/js/util/urls.js#L52-L54

Reply:
Updated.